src/main/java/org/folio/services/validator/InvoiceValidator.java

```diff
@@ -119,42 +126,76 @@ private void validateInvoiceLineFundDistributions(List<InvoiceLine> invoiceLines
       if (CollectionUtils.isEmpty(line.getFundDistributions())) {
         throw new HttpException(400, FUND_DISTRIBUTIONS_NOT_PRESENT);
       }
-
-      if (isFundDistributionSummaryNotValid(line, currencyUnit)) {
-        throw new HttpException(400, LINE_FUND_DISTRIBUTIONS_SUMMARY_MISMATCH);
-      }
+      validateFundDistributionForInvoiceLine(line.getSubTotal(),line.getFundDistributions(),currencyUnit);
     }
   }
 
-  private boolean isFundDistributionSummaryNotValid(InvoiceLine line, CurrencyUnit currencyUnit) {
-    return ObjectUtils.notEqual(sumMixedDistributions(line, currencyUnit), line.getTotal());
-  }
 
-  private Double sumMixedDistributions(InvoiceLine line, CurrencyUnit currencyUnit) {
-    return sumMixedDistributions(line.getFundDistributions(), line.getTotal(), currencyUnit);
-  }
-
-  private void validateInvoiceAdjustmentsDistributions(List<Adjustment> adjustments, CurrencyUnit currencyUnit) {
-    for (Adjustment adjustment : adjustments){
+  public static void validateInvoiceAdjustmentsDistributions(List<Adjustment> adjustments,CurrencyUnit currencyUnit) {
```

Comment:
Looks like this method is validating adjustments AND distributions but i didnt get that this method name. It seems a distribution is a child/component of the adjustment so i don't think it needs to be reflected in the method name. Adding a comment to the method is a good way to show that the adjustment and the distributions underneath are validated.

An example that illustrates what I am mean; `validateUsersFirstNameLastName`. To validate the user means to validate the first name and the last name and possibly other criteria. First name and last name dont need to be in the method name. `validateUsers` would suffice instead.

Reply:
yup , it should be fixed now.